### PR TITLE
Fhir 34606

### PR DESCRIPTION
**Pull request by Healthedata1**

### CODE REVIEW

**Comment by Healthedata1** [comment]
image

**Comment by Healthedata1** [comment]
image

File: input/includes/med-intro.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+While the relationship between Medication Statement and Medication Request is clear in principle, 
+in practice, when patients report medications, these may also be implicitly treated as statements of intent 
+about what the patient should be taking. As such, information systems may record these statements 
+as either Medication Requests or Medication Statements, or as both. The choice between these options 
+may be influenced by many factors including wording in legislation and regulations.
+
+For this reason, clients implementing the International Patient Access specification SHALL query for both 
+MedicationStatement and MedicationRequest when fetching patient Medication information.
+
+As noted their is currently no consensus how a patient can access their active, historical and future (planned) medications list. Feedback is welcome on what additional guidance can be given to promote more international alignment.
```
Comment (mrinnetmaki) [inline input/includes/med-intro.md:12]: `their` should be `there`?

Comment (isaacvetter) [inline input/includes/med-intro.md:12]:
```suggestion
As noted there is currently no consensus how a patient can access their active, historical and future (planned) medications list. Feedback is welcome on what additional guidance can be given to promote more international alignment.
```

**Pull request merged**